public/index.html

```diff
@@ -21,7 +21,7 @@
 	<!--Socket.io latest links can be found at https://cdnjs.com/libraries/socket.io-->
 </head>
 
-<body>
+<body oncontextmenu="return false">
```

Comment:
What does this do?

Reply:
Cancel context menu from appearing when right clicking the page.